gazebo_plugins/CMakeLists.txt

```diff
@@ -135,7 +129,6 @@ catkin_package(
   camera_info_manager
   std_msgs
   DEPENDS 
-    gazebo 
```

Comment:
This line had no effect anyway. `catkin_package(DEPENDS ...)` is case-sensitive, but [gazebo-config.cmake](https://bitbucket.org/osrf/gazebo/src/6fa4be17381b036e234629754cc6eea3e44a62b1/cmake/gazebo-config.cmake.in?at=default&fileviewer=file-view-default) only sets the capitalized `GAZEBO_INCLUDE_DIRS` and `GAZEBO_LIBRARIES` variables (`@PKG_NAME@` is `GAZEBO`).

Basically catkin adds `${depend_name}_INCLUDE_DIRS` and `${depend_name}_LIBRARIES` for each dependency to the exported include directories and libraries for this package (source code in [catkin_package.cmake:163](https://github.com/ros/catkin/blob/kinetic-devel/cmake/catkin_package.cmake#L163)). There will be no error or warning message because `gazebo_FOUND` is set by cmake before. Maybe this is worth a patch in catkin, like adding a check that at least one of the two variables is non-empty or making DEPENDS case insensitive by checking the lower- and upper-case variants, too (@dirk-thomas)? Or at least improve the documentation and mention the case dependency explicitly?

For `gazebo_plugins` I think that not exporting the Gazebo dependency at all is the better option because simply adding include directories and libraries would not be sufficient anyway.